core/src/test/java/com/linecorp/armeria/internal/metric/RequestMetricSupportTest.java

```diff
@@ -56,18 +51,21 @@ void httpSuccess() {
         final ClientRequestContext ctx = setupClientRequestCtx(registry);
 
         setConnectionTimings(ctx);
-        ctx.logBuilder().requestHeaders(RequestHeaders.of(HttpMethod.POST, "/foo"));
-        ctx.logBuilder().requestFirstBytesTransferred();
-        ctx.logBuilder().requestContent(null, null);
-        ctx.logBuilder().requestLength(123);
+        // FIXME(trustin): In reality, most HTTP requests will not have any name.
+        //                 As a result, `activeRequestMeterId()` will be invoked only after
```

Comment:
You mean `activeRequestMetrics.decrement()`?

Reply:
I meant `MeterIdPrefixFunction.activeRequestMeterId()`.

Reply:
Then it's `activeRequestPrefix`? 😆

Reply:
Ah yeah :-)